web-api/src/persistence/postgres/cases/getCasesByDocketNumbers.ts

```diff
@@ -312,23 +314,46 @@ async function getCaseCorrespondenceByDocketNumber(docketNumbers: string[]) {
 
 async function getHearings(
   docketNumbers: string[],
-): Promise<{ docketNumber: string; hearings: any[] }[]> {
-  const hearingsInfo = await Promise.all(
-    docketNumbers.map(async docketNumber => {
-      const hearings = await queryFull({
-        ExpressionAttributeNames: {
-          '#pk': 'pk',
-          '#sk': 'sk',
-        },
-        ExpressionAttributeValues: {
-          ':pk': `case|${docketNumber}`,
-          ':prefix': 'hearing|',
-        },
-        KeyConditionExpression: '#pk = :pk and begins_with(#sk, :prefix)',
-        applicationContext,
-      });
-      return { docketNumber, hearings };
-    }),
+): Promise<{ docketNumber: string; hearings: TrialSessionKysely[] }[]> {
+  // const hearingsInfo = await getDbReader(reader =>
+  //   reader
+  //     .selectFrom('dwTrialSessionCase as ch')
+  //     .innerJoin(
+  //       'dwTrialSession as ts',
+  //       'ch.trialSessionId',
+  //       'ts.trialSessionId',
+  //     )
+  //     .select(({ fn }) => [
+  //       'ch.docketNumber',
+  //       fn.jsonAgg('ts').as('hearings'), // This IS lying about types
+  //     ])
+  //     .where('ch.docketNumber', 'in', docketNumbers)
+  //     .groupBy('ch.docketNumber')
+  //     .execute(),
+  // );
+
+  // TODO 10493: this is a hack to make the the fromKyselyTrialSession functionality work; rethink this
```

Comment:
My recommendation: unless this is leading to clear performance issues, put a pin in it for extra credit later, probably in a followup ticket. It is more important to have everything in Postgres and then refine than to try to get the details 100% right on the first pass.